### DISCUSSION TITLE
check links from content manager

### DISCUSSION THREAD

**Discussion by dkrefta** [remarkjs/remark#1080] [General] [answered]
Hey guys there is some way that the react-markdown check when is https://something.com and show as a link?
Thanks!

**Comment by wooorm** [comment]
Hey!

a) Not everyone here is a guy :)
b) what do you mean with “check”?
c) `react-markdown` is probably not the place to check links. See the rest of the projects we work on. That’s probably where to do this

**Comment by Murderlon** [comment, marked as answer]
> Hey guys there is some way that the react-markdown check when is https://something.com and show as a link? Thanks!

I don't really understand the phrasing of the question here. Be sure to checkout [asking quality questions](https://github.com/unifiedjs/.github/blob/main/support.md#asking-quality-questions).

However, these two might help from what I'm guessing:
- https://github.com/remarkjs/remark-validate-links
- https://github.com/davidtheclark/remark-lint-no-dead-urls
- https://github.com/rehypejs/rehype-external-links

**Comment by dkrefta** [comment]
@Murderlon exactly what I was looking for <3 
 thanks for the help o//
